docs/dynamic-plugins/installing-external-plugins.md

```diff
@@ -0,0 +1,183 @@
+
+To package the plugin into a `tgz` archive, run the `npm pack` command in the `dist-dynamic` directory. This will create a `tgz` archive in the current directory that can be used to install the plugin.
+
+```bash
+cd dist-dynamic
+npm pack
+```
+
+To load the plugin from a `tgz` archive, specify the integrity hash of the archive. You can obtain the hash from the output of the `npm pack` command by using the `--json` flag.
+
+```bash
+cd dist-dynamic
+npm pack --json | head -n 10
+# If you have jq installed, use this command to get the integrity hash directly:
+# npm pack --json | jq -r '.[0].integrity'
+```
+
+To load the plugin from a `tgz` archive, host the archive on a web server accessible by your RHDH instance and specify the URL in the `dynamic-plugin-config.yaml` file.
+
+If using OpenShift, you can leverage httpd builder to serve these dynamic plugin packages:
+
+```bash
+# Pack derived dynamic plugins into `tgz` archives.
+# Repeat this step for multiple plugins (place them in the same folder).
+npm pack --pack-destination ~/test/dynamic-plugins-root/
+# Ensure you're using the same OpenShift project as the RHDH instance.
+oc project rhdh
+# Create a new build for the httpd image that will serve the `tgz` files with dynamic plugins.
+oc new-build httpd --name=plugin-registry --binary
+oc start-build plugin-registry --from-dir=dynamic-plugins-root --wait
+oc new-app --image-stream=plugin-registry
+
+## Configure your RHDH to use the dynamic plugins from the plugin-registry.
+# Edit the `dynamic-plugin-config.yaml` file:
+# plugins:
+#   - package: http://plugin-registry:8080/backstage-plugin-myplugin-1.9.6.tgz
+#   - package: http://plugin-registry:8080/backstage-plugin-myotherplugin-1.10.0.tgz
+#
+# If you add a new dynamic plugin archive in the ~/test/dynamic-plugins-root folder, and run the OpenShift build again:
+# oc start-build plugin-registry --from-dir=dynamic-plugins-root --wait
+```
+
+#### Creatign JavaScript package
+
+**Prerequisites:**
+
+- An exported derived dynamic plugin package (see: [Creating a Derived Dynamic Plugin Package](#step-1-creating-a-derived-dynamic-plugin-package)).
+
+To distribute the derived dynamic plugin package as a JavaScript package, publish it to a private or internal npm registry. Avoid using the public npm registry (npmjs.com), as the derived dynamic plugin package is structured like regular JavaScript packages but cannot be used the same way.
+
+```bash
+cd dist-dynamic
+npm publish --registry <npm_registry_url>
+```
+
+Alternatively, add the following to your `package.json` before running `npx @janus-idp/cli@latest package export-dynamic-plugin`:
+
+```json
+{
+  "publishConfig": {
+    "registry": "<npm_registry_url>"
+  }
+}
+```
+
+If you modify the `publishConfig` after running `export-dynamic-plugin`, run it again to ensure the `dist-dynamic` directory contains a `package.json` with the correct `publishConfig`.
+
+## Installing External Dynamic Plugins
```

Comment:
I wonder, should we remove (and incorporate) the information from [this section](https://github.com/janus-idp/backstage-showcase/blob/main/docs/dynamic-plugins/dynamic-plugins.md#installing-a-dynamic-plugin-package-in-the-showcase) of the `dynamic-plugins.md` document as part of this PR

Reply:
Yes, good point. We should start cleaning up the `dynamic-plugins.md`. I will add this to this PR